crates/astria-sequencer/src/transaction/checks.rs

```diff
@@ -164,16 +205,10 @@ pub(crate) async fn check_balance_for_total_fees<S: StateReadExt + 'static>(
 }
 
 fn transfer_update_fees(
```

Comment:
Why is this body updated? Probably missing something, but from a cursory glance this looks orthogonal to the new functionality?

Reply:
the function checking "fees" before was actually checking the total value used by the transaction, so it was summing fees and value transfers. however, the fees endpoint wants to return only fees used by the tx, so i had to separate these